sample/build.gradle

```diff
@@ -4,32 +4,40 @@ apply plugin: 'kotlin-parcelize'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
+    buildToolsVersion rootProject.buildToolsVersion
 
     defaultConfig {
-        minSdkVersion 24
+        minSdkVersion rootProject.minSdkVersion
```

Comment:
why the sample code need `sdkMin = 14`?

Reply:
Is this not what the library itself is using? The sample app should support the same API level that the library is supporting to actually tell if it works on said APIs.